src/core_plugins/kibana/public/visualize/editor/agg_params.js

```diff
@@ -81,10 +81,9 @@ uiModules
         // build collection of agg params html
         type.params.forEach(function (param, i) {
           let aggParam;
+
           // if field param exists, compute allowed fields
-          if (param.name === 'field') {
```

Comment:
What's the reason for removing the conditional here? This seems to alter the flow of the loop so that if there are no `indexedFields` only the first param will ever be rendered, whereas before all params up to the `field` param would be rendered.